guides/common/modules/proc_configuring-kerberos-authentication-for-remote-execution.adoc

```diff
@@ -1,7 +1,7 @@
 [id="configuring-kerberos-authentication-for-remote-execution_{context}"]
 = Configuring Kerberos Authentication for Remote Execution
 
-You can use Kerberos authentication to establish an SSH connection for remote execution on {Project} hosts.
+You can use Kerberos authentication to establish a Script connection for remote execution on {Project} hosts.
```

Comment:
All the references to the underlying technology should stay. Even if you're running a script, the connection itself is ssh

Reply:
This is still unresolved.